yattag/indentation.py

```diff
@@ -11,34 +17,39 @@
 
 class TokenMeta(type):
 
-    _token_classes = {}
+    _token_classes = {}  # type: Dict[str, Any]
 
     def __new__(cls, name, bases, attrs):
+        # type: (str, Tuple[Any], Dict[str, Any]) -> Any
         kls = type.__new__(cls, name, bases, attrs)
         cls._token_classes[name] = kls
         return kls
 
     @classmethod
     def getclass(cls, name):
+        # type: (str) -> Any
         return cls._token_classes[name]
 
 # need to proceed that way for Python 2/3 compatility:
-TokenBase = TokenMeta('TokenBase', (object,), {})
+TokenBase = TokenMeta('TokenBase', (object,), {}) # type: Any
 
-class Token(TokenBase):
-    regex = None
+class Token(TokenBase): # type: ignore
```

Comment:
````
error: Class cannot subclass 'TokenBase' (has type 'Any')
````

is the error message you get without it. And yes, it's because the current metaclass code is not done in a way so that mypy's static analysis can understand it. (At least that's how I understand it; I haven't used mypy with metaclasses in my projects so far, any my code is typically just python3-only, which simplifies the situation further.)